src/main/java/core/worldEntities/types/characters/Character.java

```diff
@@ -20,17 +24,31 @@
      * */
 
     private String team;
-    private int health;
+    private float health;
     private int level;
     private ArrayList<Item> inventory;
     private Class<? extends CharacterInputDevice> inputDeviceType;
+    private ArrayList<CollisionBehaviour<Character, ?>> collisionBehaviours;
 
     public Character(WorldEntityManager entityManager, BodyDef bodyDef, FixtureDef... fixtureDefs) {
         super(entityManager, bodyDef, fixtureDefs);
         this.team = "";
         this.health = 0;
         this.level = 0;
         this.inventory = new ArrayList<>();
+
+        this.collisionBehaviours = new ArrayList<>();
```

Comment:
I think this should be moved to the character manager, as it is manipulating the entity

Reply:
i think it makes sense to define the `CollisionBehaviour` in the manager, but maybe keep this array in the entity since different characters might need collision behaviours for more than just damage

this is a good question for Ben